packages/icons/scripts/optimize.js

```diff
@@ -0,0 +1,107 @@
+const { optimize: svgoOptimize } = require("svgo");
```

Comment:
"optimize" is not specific, it could optimize a lot of things, maybe rename to "optimize-svg" ?